hathor/manager.py

```diff
@@ -233,6 +227,13 @@ def __init__(self,
         self.lc_check_sync_state.clock = self.reactor
         self.lc_check_sync_state_interval = self.CHECK_SYNC_STATE_INTERVAL
 
+    def get_default_capabilities(self) -> list[str]:
```

Comment:
Missing docstring.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/787/commits/914dd775149b55db26cf5e9d0e8e29d3ef118681

Reply:
Thanks!